Lib/Container/ContainerBuilder.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace CakePHPUtil\Lib\Container;
+
+class ContainerBuilder {
+
+	protected static $_container;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "$_container" to match the regular expression ^[a-z][a-zA-Z0-9]*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS116)